external-crates/move/crates/move-compiler/src/shared/ide.rs

```diff
@@ -245,7 +251,10 @@ impl
         {
             match entry {
                 MemberEntry::Member(mident, name) => {
-                    members.insert((*symbol, *mident, *name));
+                    members
+                        .entry((*mident, name.value))
+                        .or_default()
+                        .insert(*symbol);
```

Comment:
What is the motivation for this change in behavior?

Reply:
This is related to the change in data representation. When searching for auto-imports, I need to look up in-scope members by module identifier and member name (to filter out in-scope members from the auto-imports list). Previously these were represented by a set with an entry being a triple: module identifier, member name, member alias. I changed it to a map with the tuple consisting of module identifier and member name being the key. As a result, modification of this data structure had to change as well